prestoadmin/package.py

```diff
@@ -93,9 +93,15 @@ def rpm_install(rpm_name):
     if env.nodeps:
         nodeps = '--nodeps '
 
-    ret = sudo('rpm -i %s%s' %
-               (nodeps, os.path.join(constants.REMOTE_PACKAGES_PATH,
-                                     rpm_name)))
+    if env.java8_home is None:
```

Comment:
you can check instead if the java8_home key is in the dictionary (see http://stackoverflow.com/questions/1602934/check-if-a-given-key-already-exists-in-a-dictionary)